### PR TITLE
fix non-exhaustive pattern matching

### PR DESCRIPTION
**Pull request by gabriele83**
added some improvements

### CODE REVIEW

**Comment by gabriele83** [comment]
We'd be happy to contribute with testing, or help in any other way if necessary.

File: migrator/src/main/scala/akka/persistence/jdbc/migrator/JournalMigrator.scala [hunk 1]
```diff
@@ -121,23 +122,11 @@ final case class JournalMigrator(profile: JdbcProfile)(implicit system: ActorSys
     } yield Done
   }
 
-  /**
-   * Unpack a PersistentRepr into a PersistentRepr and set of tags.
-   * It returns a tuple containing the PersistentRepr and its set of tags
-   *
-   * @param pr the given PersistentRepr
-   */
-  private def unpackPersistentRepr(pr: PersistentRepr): (PersistentRepr, Set[String]) = {
```
Comment (Tochemey) [inline migrator/src/main/scala/akka/persistence/jdbc/migrator/JournalMigrator.scala:130]: Any reason why you remove this method. I don't remember how it was used but I do know that we need to unpack the events with tags and without tags.

**Review by Tochemey — COMMENTED** [review]
Good job.

File: migrator/src/main/scala/akka/persistence/jdbc/migrator/JournalMigrator.scala (hunk 1 above)
Comment (gabriele83) [reply migrator/src/main/scala/akka/persistence/jdbc/migrator/JournalMigrator.scala:130]: I removed it because it was not used

**Pull request merged**